packages/cli/src/commands/console.ts

```diff
@@ -12,7 +12,15 @@ export class Console extends Command {
     useColors: true,
   }
 
+  static flags = {
+    help: flags.help({char: "h"}),
+  }
+
   async run() {
+    // @ts-ignore noUnusedLocals
+    // This is needed to make sure that help flag is working correctly
+    const {flags} = this.parse(Console)
```

Comment:
Again, all you need is `this.parse(Console)`

Reply:
Done 👍